fuel-vm/src/tests/blockchain.rs

```diff
@@ -2261,3 +2261,444 @@ fn various_ldc_issues_poc() {
         }
     });
 }
+
+// === Begin temporary tests for ldcv2 ===
```

Comment:
Since the LDCv2 isn't supposed to exist for a long time, these tests are temporary. We're planning to migrate to a separate opcode instead of overloading ldc like this. This section should be removed when we separate the instructions, which is likely before merging this PR.

Reply:
Hmm, I think we can remove it now, it seems old LDC for blobs will exist along with new LDCv2 that we plan to use for contracts. I don't like it, but because we need a backward compatibility with the testnetn, we can't drop it when we move do mainnet=(